### DISCUSSION TITLE
List of all supported colors?

### DISCUSSION THREAD

**Discussion by hexiro** [Textualize/rich#2261] [Q&A] [answered]
Hello! is there is a place where I can find the list of all colors that can be used by default inside block like such: `[green]text[/green]`?

**Comment by hexiro** [comment]
This looks like what I was looking for although I'm not sure if this exists somewhere in the docs where it'd be easier to find and look at.

https://github.com/Textualize/rich/blob/2c93dce92788cec32a39de14bf54a51188284cd8/rich/color.py#L46

**Comment by willmcgugan** [comment, marked as answer]
https://rich.readthedocs.io/en/latest/appendix/colors.html#appendix-colors

**Reply by hexiro** [reply]
thank you very much!
